.eslintrc.js

```diff
@@ -113,7 +113,7 @@ module.exports = {
         'no-shadow': 2,
 
         // Vue
-        'vue/multi-word-component-names': 1,
+        'vue/multi-word-component-names': 0,
```

Comment:
@ann-kilzer @RageZBla can we keep this as a warning? 
It's a good practice and we should try to enforce it, however, I moved to warning temporarily so we can pass build with our current code. We need to update our component names to follow vue standards (I'll create a ticket for this)

Reply:
@ermish thanks for the extra context. I have re-enabled the lint rule.